From 5e7ce27626a44428c01e0e5ab3fe60ef98ca788c Mon Sep 17 00:00:00 2001
From: Petr Tesarik <ptesarik@suse.cz>
Date: Mon, 26 Apr 2021 15:29:51 +0200
Subject: kexec-tools: Remove duplicate definition of ramdisk

The ramdisk variable is defined in kexec/arch/ppc/kexec-ppc.c. This
other definition is not needed and breaks build with -fno-common.

Signed-off-by: Petr Tesarik <ptesarik@suse.com>
Signed-off-by: Simon Horman <horms@verge.net.au>
[Retrieved from:
https://git.kernel.org/pub/scm/utils/kernel/kexec/kexec-tools.git/commit/?id=5e7ce27626a44428c01e0e5ab3fe60ef98ca788c]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 kexec/arch/ppc/kexec-elf-ppc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kexec/arch/ppc/kexec-elf-ppc.c b/kexec/arch/ppc/kexec-elf-ppc.c
index ad43ad1..4a4886e 100644
--- a/kexec/arch/ppc/kexec-elf-ppc.c
+++ b/kexec/arch/ppc/kexec-elf-ppc.c
@@ -33,7 +33,6 @@
 static const int probe_debug = 0;
 
 unsigned char reuse_initrd;
-const char *ramdisk;
 int create_flatten_tree(struct kexec_info *, unsigned char **, unsigned long *,
 			char *);
 
-- 
cgit 1.2.3-1.el7

